test/parallel/test-fs-readfile-empty.js

```diff
@@ -27,20 +27,21 @@ require('../common');
 const assert = require('assert');
 const fs = require('fs');
 const fixtures = require('../common/fixtures');
+const { mustCall } = require('../common')
```

Comment:
Hi, missing semicolon here:
```suggestion
const { mustCall } = require('../common');
```


Besides, remember to remove the duplicated `require('..common');` in line 23 above if you `require('../common')` here.